cpp/src/dataframe/cudf_table.cuh

```diff
@@ -477,6 +438,46 @@ public:
 
   };
 
+  __device__
+  void* get_column_device_pointer(size_type column_index)
```

Comment:
No longer necessary?

Reply:
no. you want it removed?

Reply:
Yeah, no reason to have unused code laying around if we can avoid it.